lib/reporter.js

```diff
@@ -93,12 +95,18 @@ var CoverageReporter = function(rootConfig, helper, logger) {
     if (browsers) {
       browsers.forEach(function(browser) {
```

Comment:
Seems karma@0.10 did not have method `onBrowserStart`.
For removing duplication we can do small refactoring:

```js
browser.forEach(this.onBrowserStart.bind(this));
```